packages/shared-ux/page/kibana_no_data/impl/src/kibana_no_data_page.tsx

```diff
@@ -46,9 +46,5 @@ export const KibanaNoDataPage = ({ onDataViewCreated, noDataConfig }: KibanaNoDa
     return <NoDataViewsPrompt onDataViewCreated={onDataViewCreated} />;
   }
 
-  if (!dataExists) {
-    return <NoDataConfigPage noDataConfig={noDataConfig} />;
-  }
-
-  return null;
+  return <NoDataConfigPage noDataConfig={noDataConfig} />;
```

Comment:
I think it's safer not to return null, but to show `NoDataConfigPage` as default.